Tasks/AzureRmWebAppDeployment/package.json

```diff
@@ -17,6 +17,8 @@
   },
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "dependencies": {
-    "q": "1.4.1"
+    "q": "1.4.1",
+    "xml2js": "0.4.13",
+    "uuid": "3.1.0"
```

Comment:
Please ensure appropriate attributions are there for uuid in TPN. I have sent you a mail on this regard.

Reply:
UUID notice alreay exists in TPN.